inbox/api/wsgi.py

```diff
@@ -26,7 +22,7 @@ def init_process(self):
 
         configure_logging(log_level=LOGLEVEL)
 
-        if MAX_BLOCKING_TIME:
+        if config.get("USE_GEVENT", True) and MAX_BLOCKING_TIME:
```

Comment:
What does it mean if `USE_GEVENT` is True, but we're in a `ThreadWorker` and gevent monkey patches are no longer applied?

Reply:
@mrhiggi-close I removed Tracer entirely (since it's only used from API pods) and also ensured that the invalid combination is now impossible in [`c617f97` (#902)](https://github.com/closeio/sync-engine/pull/902/commits/c617f97ec12250f4b90eacf4cad4b2c7a0d4a6bc)